kentik_api_library/kentik_api/public/__init__.py

```diff
@@ -38,3 +38,17 @@
     SecondaryIP,
     VRFAttributes,
 )
+from .errors import (
```

Comment:
Could you explain why are we doing it?

Reply:
To allow user to use our library defined errors in try except blocks

Reply:
Ok, so this is a part of importing chain required for following import to work `from kentik_api import AuthError`
I missed that during integration tests. I used there a direct import: `kentik_api.public.errors import AuthError`, which doesn't test the whole import chain. I'll make a note about it.
Resolved.